packages/vm/src/vm.ts

```diff
@@ -231,7 +232,7 @@ export class VM {
       blockchain,
       stateManager,
     }
-    const evmCopy = new EVM(evmOpts)
+    const evmCopy = new EVM(evmOpts) // TODO fixme (should copy the EVMInterface, not default EVM)
```

Comment:
Imo we could do this in this PR by just adding a `copy` method to the EVMInterface, and implemting it in EVM in the way you've done here. Then replace this line with `evmCopy = evm.copy()`. What do you think?

Reply:
I think we should keep this as a TODO for now (?)

Reply:
Ah sure yes, I thought this was a simple implementation but since it's more complicated than that let's leave for another PR :)